core/device_hooks/common_kernels.inc.cpp

```diff
@@ -66,6 +68,22 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 namespace gko {
 namespace kernels {
 namespace GKO_HOOK_MODULE {
+
+
+template <typename SourceType, typename TargetType>
+GKO_DECLARE_CONVERT_PRECISION_KERNEL(SourceType, TargetType)
+GKO_NOT_COMPILED(GKO_HOOK_MODULE);
+GKO_INSTANTIATE_FOR_EACH_VALUE_CONVERSION(GKO_DECLARE_CONVERT_PRECISION_KERNEL);
+
+
+template <typename IndexType>
+GKO_DECLARE_PREFIX_SUM_KERNEL(IndexType)
+GKO_NOT_COMPILED(GKO_HOOK_MODULE);
+GKO_INSTANTIATE_FOR_EACH_INDEX_TYPE(GKO_DECLARE_PREFIX_SUM_KERNEL);
+// explicitly instantiate for size_type, as this is used in the SellP format
+template GKO_DECLARE_PREFIX_SUM_KERNEL(size_type);
```

Comment:
Should these be in a namespace?

Reply:
Yes, I'll probably move them into a `precision_conversion` namespace. Not sure about `prefix_sum` though. That one I forgot to add to the `common_kernels`, but the name is already kind of self-explanatory. Do we need to put it into a namespace?